addons/mail/static/src/discuss/core/public_web/discuss_search.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.DiscussSearch">
+        <div t-if="tagsList.length" class="d-flex flex-wrap gap-1 m-2">
+            <TagsList tags="tagsList"/>
+        </div>
+        <div t-ref="searchBox" class="o-mail-DiscussSearch d-flex flex-grow-1 align-items-center justify-content-center rounded gap-1" t-att-class="{ 'o-active': state.searchValue.length }">
+            <input class="form-control rounded-3 border-0" t-att-placeholder="inputPlaceholder" t-model="state.searchValue" t-ref="searchInput" t-on-keydown="onKeydownInput"/>
+            <button t-if="state.searchValue.length" class="btn btn-light p-1 rounded-0" t-on-click="() => state.searchValue = ''"><i class="oi oi-close" title="Clear quick search"/></button>
+        </div>
+        <NavigableList t-props="navigableListProps"/>
+    </t>
+
+    <t t-name="discuss.DiscussSearch.partner">
+        <div class="bg-inherit position-relative d-flex flex-shrink-0" style="width:26px;height:26px">
+            <img class="w-100 h-100 rounded" t-att-src="option.partner.avatarUrl" alt="Thread Image"/>
+        </div>
+        <span class="mx-2 text-truncate" t-esc="option.partner.name" />
+    </t>
+
+    <t t-name="discuss.DiscussSearch.channel">
+        <div class="bg-inherit position-relative d-flex flex-shrink-0" style="width:26px;height:26px">
+            <img class="w-100 h-100 rounded" t-att-src="option.channel.avatarUrl" alt="Thread Image"/>
+        </div>
+        <span class="mx-2 text-truncate" t-esc="option.channel.displayName" />
+    </t>
+
+    <t t-name="discuss.DiscussSearch.new">
+        <span class="badge rounded-pill text-bg-primary">CREATE</span>
```

Comment:
The visual of "create" is too much, this looks like an avatar. I would remove and just have text, no rounded-pill badge.